FlowCrypt/Models/Realm Models/RecipientRealmObject.swift

```diff
@@ -9,21 +9,11 @@
 import Foundation
 import RealmSwift
 
-final class LongId: Object {
```

Comment:
@tomholub 
I checked source code and we don't use this type. Maybe I'm wrong and you need to keep this one

Reply:
If we don't use it, throw it out :+1: